.travis.yml

```diff
@@ -1,15 +1,3 @@
 language: java
-jdk:
-  - oraclejdk8
```

Comment:
By removing this, Travis switches to Java11.  Gradle still works with Java8.  ~~I couldn't, yet, figure out why this PR doesn't build with Java8 on Travis... it does build with Java8 locally, and I the master branch builds with Java8 on Travis.~~  Specifying `openjdk8` here works for Travis.

Reply:
switch to openjdk